x-pack/plugins/lists/server/services/exception_lists/create_endpoint_trusted_apps_list.ts

```diff
@@ -0,0 +1,77 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { SavedObjectsClientContract } from 'kibana/server';
+import uuid from 'uuid';
+
+import {
+  ENDPOINT_TRUSTED_APPS_LIST_DESCRIPTION,
+  ENDPOINT_TRUSTED_APPS_LIST_ID,
+  ENDPOINT_TRUSTED_APPS_LIST_NAME,
+} from '../../../common/constants';
+import { ExceptionListSchema, ExceptionListSoSchema, Version } from '../../../common/schemas';
+
+import { getSavedObjectType, transformSavedObjectToExceptionList } from './utils';
+
+interface CreateEndpointListOptions {
+  savedObjectsClient: SavedObjectsClientContract;
+  user: string;
+  tieBreaker?: string;
+  version: Version;
+}
+
+/**
+ * Creates the Endpoint Trusted Apps agnostic list if it does not yet exist
+ *
+ * @param savedObjectsClient
+ * @param user
+ * @param tieBreaker
+ * @param version
+ */
+export const createEndpointTrustedAppsList = async ({
+  savedObjectsClient,
+  user,
+  tieBreaker,
+  version,
+}: CreateEndpointListOptions): Promise<ExceptionListSchema | null> => {
+  const savedObjectType = getSavedObjectType({ namespaceType: 'agnostic' });
+  const dateNow = new Date().toISOString();
+  try {
+    const savedObject = await savedObjectsClient.create<ExceptionListSoSchema>(
+      savedObjectType,
+      {
+        _tags: [],
```

Comment:
Should this have any tags? Seems like we could add a few for internal use

Reply:
Not sure 🤷‍♂️ 
I don't think I saw any in the Exceptions list definition, but the Items do have it. Let me know you think should be there 🙂 

That's actually a question of mine when we get to the "Create" flow for trusted apps. We will be capturing those per OS, and in looking at the Exceptions items, it seems to store the OS in the `_tags[]` - do I have that that right?

Reply:
We're adding a separate field for OS most likely... it's not great to rely on _tags for that. There's a ticket for it i believe, let me find it.

Reply:
(We can change them all at once though, so I think this would be fine for now).

Reply:
Sounds good 👍 . Thanks @madirey